inorbit_connector/connector.py

```diff
@@ -160,6 +163,35 @@ def handler_wrapper(command_name: str, args: list, options: dict):
 
         self._robot_session.register_command_callback(handler_wrapper)
 
+    def _register_builtin_command_handlers(self) -> None:
+        """Register built-in command handlers that all connectors need."""
+
+        def get_state_handler(command_name: str, args: list, options: dict):
+            """Handle get_state command from InOrbit."""
+            if command_name == "get_state":

```

Comment:
I think this belongs to the edge-sdk, is partially implemented already for loading/unloading modules.